pkg/cmd/server/start/master_args.go

```diff
@@ -188,6 +189,15 @@ func (args MasterArgs) BuildSerializeableMasterConfig() (*configapi.MasterConfig
 
 	dnsServingInfo := servingInfoForAddr(&dnsBindAddr)
 
+	var clusterNetworkConfig []configapi.ClusterNetworkEntry
+
+	for _, cidr := range strings.Split(args.NetworkArgs.ClusterNetworkCIDR, ",") {
+		clusterNetworkEntry := configapi.ClusterNetworkEntry{
+			ClusterNetworkCIDR: cidr,
+		}
+		clusterNetworkConfig = append(clusterNetworkConfig, clusterNetworkEntry)
+	}
+
```

Comment:
My understanding is command line arguments is for simple/essential use cases and complex use cases need to be specified in configuration file. If we treat multiple cidr use case as complex then we don't need to support command line args. 
Supporting comma separated list here makes it harder to specify different hostsubnetLength for each cidr later.